README.md

```diff
@@ -111,33 +111,44 @@ Reads the configuration and starts relaying packets.
      git checkout v0.15.1
      make install
      ```
+2. Make sure `iris` binary is installed on your system
 
-2. Create a new account and fund it
+   - you must be running Linux or OSX on amd64
+   - [install Go 1.15+](https://golang.org/doc/install) and ensure that `$PATH` includes Go binaries (you may need to restart your terminal session)
+   - clone and install `iris`:
+     ```sh
+     git clone https://github.com/irisnet/irishub
+     cd irishub
+     git checkout v1.1.1
+     make install
+     ```
+
+3. Create a new account and fund it
 
    ```sh
    wasmd keys add sender
    JSON=$(jq -n --arg addr $(wasmd keys show -a sender) '{"denom":"usponge","address":$addr}')
    curl -X POST --header "Content-Type: application/json" --data "$JSON" https://faucet.oysternet.cosmwasm.com/credit
    ```
 
-3. Create a valid IrisNet address to send tokens to
+4. Create a valid IRISnet address to send tokens to
 
    ```sh
-   RCPT=$(ibc-setup keys list | grep nyancat | cut -d' '  -f2)
+   iris keys add receiver
    ```
 
-   TODO: setup irisnet binary and make real account there
+   [Get testnet tokens](https://github.com/irisnet/testnets/tree/master/nyancat#faucet) if you want to send tokens to `oysternet`.
```

Comment:
Note, you can only use the faucet once per 24 hours, so we cannot fund the relayer and the receiver in the same day. Maybe a comment here on that, or they just get the error message.

Reply:
Thanks for your note, we will increase the limit to 2 or more. ❤️